nefertari_sqla/documents.py

```diff
@@ -134,8 +142,11 @@ def get_es_mapping(cls):
             properties[name] = TYPES_MAP[column_type]
 
         for name, column in relationships.items():
-            if name in cls._nested_relationships:
+            if name in cls._nested_relationships and not depth_reached:
                 column_type = {'type': 'object'}
```

Comment:
I am wondering why the type is `object` and not `nested`. I thought that we were switching to nested object mapping. https://www.elastic.co/guide/en/elasticsearch/guide/current/nested-mapping.html